src/js/add_input_icon.js

```diff
@@ -227,6 +227,8 @@ async function addRelayIconToInput(emailInput) {
     // Create "Get unlimited aliases" button
     const getUnlimitedAliasesBtn = createElementWithClassList("a", "fx-relay-menu-get-unlimited-aliases");
     getUnlimitedAliasesBtn.textContent = browser.i18n.getMessage("popupGetUnlimitedAliases");
+    getUnlimitedAliasesBtn.setAttribute("target", "_blank");
+    getUnlimitedAliasesBtn.setAttribute("rel", "noopener noreferrer");
```

Comment:
Since it's not super obvious why this attribute needs to be added, it could be helpful for future readers to add a link to an explanatory article, e.g. https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/.